source/user/debugging.rst

```diff
@@ -13,4 +13,18 @@ There are two types of review dependencies: implicit and explicit.
 
 **Implicit dependency** is a dependency introduced by the git tree structure - if ``review A`` is based on a parent commit from ``review B`` and ``review B`` is not merged yet, ``review B`` automatically becomes a dependency of ``review A``.
 
-**Explicit dependency** is a dependency specified in git commit message by ``Depends-On: $Change-ID``.
+**Explicit dependency** is a dependency specified in git commit message by ``Depends-On: $Change-ID``
+
+Unittest failures
+-----------------
+
+Logs
+````
+You should probably start your debugging with checking what tests failed - you can do this in ``scons_test-FAIL.log.gz``. You can look for the tests and their results in ``job-output.txt.gz`` looking for the test name. 
+
+If this doesn't help you, try looking for ``scons: *`` to see all errors raised by SCons - most often failures will be precedented by Python stacktrace.
+
+
+Flakiness
+`````````
+Sometimes it's just test flakiness and not much you can do. To run the tests again, type ``recheck`` in a comment in your review.
```

Comment:
Add some information about possible further steps:
- letting know about a broken/flaky test on the #ci channel (tungstenfabric slack)
- contacting the test author, if known